streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -86,41 +85,51 @@ void removeAllSensors() {
         }
     }
 
-    /**
-     * Create {@link StreamTask} with its assigned partitions
-     *
-     * @param id              the ID of this task
-     * @param partitions      the collection of assigned {@link TopicPartition}
-     * @param topology        the instance of {@link ProcessorTopology}
-     * @param consumer        the instance of {@link Consumer}
-     * @param changelogReader the instance of {@link ChangelogReader} used for restoring state
-     * @param config          the {@link StreamsConfig} specified by the user
-     * @param metrics         the {@link StreamsMetrics} created by the thread
-     * @param stateDirectory  the {@link StateDirectory} created by the thread
-     * @param cache           the {@link ThreadCache} created by the thread
-     * @param time            the system {@link Time} of the thread
-     * @param producer        the instance of {@link Producer} used to produce records
-     */
```

Comment:
This is an internal class, and this javadoc doesn't say anything that the method signature doesn't say. I added a new constructor and changed the existing one, so I just removed the doc rather than updating it.